notebooks/development/Build Breaker identification - PIP.ipynb

```diff
@@ -0,0 +1,1603 @@
+{
```

Comment:
<p>I like this visualization. Could you wrap the whole build-breaker analysis into a single function which accepts log and extracts following information out of it?</p><p><br></p><ul><li>all dependencies with versions which were requested to install (thoth-lab in this case - root node)</li><li>all dependencies with versions which were tried to be installed (red and green dots)</li><li>dependencies with versions which were installed (green dots)</li><li>dependency with version which failed the installation (certifi in this case)</li><li>parent with version for the failed dependency (requests in this case)</li><li>error message you extracted from the log</li></ul>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>I hope I can. I'd like to have a more generic way which would work for pipenv, there is a bit more experimenting required in that direction, but at least for pip, we can do that.</p>